src/sentry/api/serializers/models/auditlogentry.py

```diff
@@ -16,6 +18,14 @@ def fix(data):
     return data
 
 
+def override_actor_id(user):
```

Comment:
if we're only doing this for the SCIM integration we'll want to match on that prefix as well

Reply:
we should override the actor_id for all integrations that have a hash? i had a custom integration and the audit log also showed up with the super long name

Reply:
i think this is a good idea -- although longer term we may just want to create app installations with a human readable name for the audit log, so maybe we do this short term then make another backlog ticket to tackle this fully (not sure how many different integration types there are, could get unwieldy)